### DISCUSSION TITLE
下载部署文件时magisk的url一直拒绝访问且一直产生logs文件是怎么回事

### DISCUSSION THREAD

**Discussion by badnng** [LSPosed/MagiskOnWSALocal#360] [Q&A] [answered]
已经尝试自己下载到环境上面，然后改文件名进行重新尝试，就一直[NOTICE] CUID#10 - Redirecting to https://raw.githubusercontent.com/topjohnwu/magisk-files/25.2/app-release.apk
我在想，是不识别还是怎么的。。

**Comment by Howard20181** [comment, marked as answer]
挂代理

**Reply by badnng** [reply]
我已经在自己电脑上下载传过去了，但是它不识别啊还是会继续下

**Reply by Howard20181** [reply]
如果全部文件都下载完了用离线模式就好
